mscore/downloadUtils.h

```diff
@@ -15,34 +15,39 @@
 
 namespace Ms {
 
+
+/// Class for managing downloads
+/// Warning: For this class to show a progress dialog, it needs a QWidget as parent
+
 class DownloadUtils : public QObject
       {
       Q_OBJECT
 
-      QByteArray sdata;
+      QByteArray sdata; /// the final data received
```

Comment:
If you intend these comments to work with Doxygen you should rather use the following format (or some other of those listed [here](http://www.doxygen.nl/manual/docblocks.html#memberdoc)):
```
///< the final data received
```

Otherwise they will likely be assigned to the next member variable instead of the one in this line.